src/modules/main/components/WelcomeSection/WelcomeSection.jsx

```diff
@@ -0,0 +1,21 @@
+import styles from "./welcome_section.module.scss";
+import dog from "../../../../assets/image/encourages/dog.png";
```

Comment:
Wouldn't it be easier to call it `welcomeSectionImage`? I think it will be more verbose in this case